### ISSUE TITLE
StopBits failing

### ISSUE THREAD

**Issue by minborg** [OpenHFT/Chronicle-Bytes#176]
The stop bits fails as demonstrated by the following test:

```
public class StopBitTest {

    @Test
    public void testStopBit() {

        final String s = IntStream.range(0, 100)
                .mapToObj(Integer::toString)
                .collect(Collectors.joining());

        final Bytes<byte[]> bytes = Bytes.from(s);

        final Bytes<ByteBuffer> b = Bytes.elasticByteBuffer();

        b.write8bit(bytes);

        Assert.assertEquals(s, b.read8bit());

    }

}

```

**Comment by RobAustin** [comment]
@minborg please prioritise as this relates to support issue https://support.chronicle.software/staff/ticket/48#340 

Thanks.

**Comment by JerryShea** [comment]
I believe that the regression was introduced in https://github.com/OpenHFT/Chronicle-Bytes/commit/420ea2a65d88fc2661a898a5bd25ff7b4efb1a27 and released in bytes 2.21ea13 chronicle-bom-2.21ea75
